hw10/index.js

```diff
@@ -0,0 +1,126 @@
+window.addEventListener('DOMContentLoaded', () => {
+
+    //task1
+    let task1 = 'Вивести на сторінку в один рядок через кому числа від 10 до 20: ';
+    for (let i = 10; i <= 20; i++) {
+        const addString = i === 20 ? `${i};` : `${i},`;
```

Comment:
it will be better to create a variable for `20`, it will be easier to update in a case `change right border`